test/bats/tests/gcp/gcp_v1_secretproviderclass.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   provider: gcp
   parameters:
-    auth: provider-adc
+    #auth: provider-adc
```

Comment:
Can we please add a comment with an explanation of why we need to comment this line? It would help during future rebases.